inductiva/tasks/task.py

```diff
@@ -227,10 +228,10 @@ def download_outputs(
         # the size of the file, etc.)
         response = api_response.response
 
+        output_dir = files.resolve_path(output_dir)
```

Comment:
Are we informing the user of what is going on? Downloading can take forever, right?
How much data did we downloaded? Are we informing the user? Are we informing the user of how much space he has left on his local device?

Do you see my point here? :)

Reply:
This already gives information about the space the dir occupies and the time it took to download. Do you want me to add the space left on the device? 
<img width="490" alt="image" src="https://github.com/inductiva/inductiva/assets/114397668/e98f40ce-76e9-4ec9-a2f6-6fe2e7a00105">